src/uu/cksum/src/cksum.rs

```diff
@@ -380,7 +381,11 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
     let untagged: bool = matches.get_flag(options::UNTAGGED);
     let tag: bool = matches.get_flag(options::TAG);
 
-    let binary = if untagged && tag {
+    let args: Vec<String> = std::env::args().collect();
+    let binary = if args.contains(&"--tag".to_string()) && args.contains(&"--untagged".to_string())
+    {
+        // ugly workaround to detect if the two arguments have been passed at the same time
+        // clap with overrides_with
         false
```

Comment:
This case cannot happen, because `overrides_with` was used. Is this perhaps an outdated comment that you forgot?

Reply:
i updated it to
```

    let args: Vec<String> = std::env::args().collect();
    let binary = if args.contains(&"--tag".to_string()) && args.contains(&"--untagged".to_string())
```